source/transactions/transactions.rst

```diff
@@ -982,6 +982,15 @@ label. For example:
                 continue
             raise
 
+Handling command errors
+-----------------------
+
+Drivers MUST document that command errors that do not have the
```

Comment:
I think this should just be "command errors may abort the transaction on the server", regardless of the label. the server also aborts the transaction on many transient transaction errors (e.g. WriteConflict).

Reply:
Should we recommend re-raising all command errors inside `with_transaction` callback then? Since we do not have a reliable indicator which command aborts the transaction and with does not.

Reply:
I guess the distinction is perhaps relevant in that, if the user _doesn't_ re-raise an error but it's a TransientTransactionError, the driver still ends up doing the correct thing and retrying, since it gets NoSuchTransaction, a different TransientTransactionError.
whereas for a non-transient error we end up with the infinite loop issue. 

so I think it's less strictly necessary to recommend re-raising TransientTransactionErrors. however, the reason it happens to work in that case is a little funny. so for simplicity maybe it's best to not distinguish so users don't have to think about error labels and just say re-raise all errors?

Reply:
Good point, I agree. We should recommend re-raising all errors. If a user has some specific use case, core API will be a better choice.